mslib/msui/msui_web_browser.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, url: str):
         self._url = url
         self.profile = QWebEngineProfile().defaultProfile()
         self.profile.setPersistentCookiesPolicy(QWebEngineProfile.ForcePersistentCookies)
-        self.browser_storage_folder = os.path.join(MSUI_CONFIG_PATH, '.cookies')
+        self.browser_storage_folder = os.path.join(MSUI_CONFIG_PATH, 'webbrowser/.cookies')
```

Comment:
```suggestion
        self.browser_storage_folder = os.path.join(MSUI_CONFIG_PATH, 'webbrowser', '.cookies')
```

`os.path.join` can take a variable number of arguments and inserts the proper path separator between all of them. Since the path separator on windows wouldn't be `/` (I think) it shouldn't be hardcoded here.

Reply:
Thank you :)